src/crimson/os/seastore/seastore_types.cc

```diff
@@ -176,16 +201,16 @@ std::ostream &operator<<(std::ostream &out, const paddr_list_t &rhs)
   return out << ']';
 }
 
-std::ostream &operator<<(std::ostream &lhs, const delta_info_t &rhs)
+std::ostream &operator<<(std::ostream& out, const delta_info_t& delta)
```

Comment:
std::ostream &out, const delta_info_t &delta

Reply:
done, and fixed the others

Reply:
I've got opposite placement suggestions previously.

Do we have a strong preference about & placement in Ceph? Or it is OK as long as the style is consistent in the same file?

Reply:
Most code has been written with the &/* on the variable name, so let's just stick with that.